### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by mobinasadi** [tweepy/tweepy#1274] [Questions] [answered]
Can a user automatically check messages to a Twitter user?
Please help me

**Comment by Harmon758** [comment, marked as answer]
I'm not sure what you mean by automatically, but you can use [`API.list_direct_messages`](https://tweepy.readthedocs.io/en/latest/api.html#API.list_direct_messages) to check all DMs sent and received within the last 30 days, and then filter them by recipient ID.
Also see https://developer.twitter.com/en/docs/direct-messages/sending-and-receiving/api-reference/list-events.
